### ISSUE TITLE
Weird drop in predictions

### ISSUE THREAD

**Issue by ricokoff** [zhouhaoyi/Informer2020#348]
Hi,

I am trying to get your model to work with financial data for my Thesis. However, the predictions seem to be shifted by quite a bit. I added an image to show this. Do you have any idea what could cause this shift? I am currently doing multivariate predict univariate (MS) with 6 features, including the target.
[prediction.pdf](https://github.com/zhouhaoyi/Informer2020/files/9040308/prediction.pdf)

Some of my parameters are: 
--features MS 
--target close 
--freq h 
--seq_len 96 
--label_len 48
--pred_len 96
--enc_in 6 
--dec_in 6 
--c_out 1 
--d_model 512 
--n_heads 8 
--e_layers 2 
--d_layers 1 
--d_ff 2048 
--dropout 0.05
--embed timeF 
--activation gelu 
--num_workers 10 
--itr 2 
--train_epochs 10 
--batch_size 32 
--patience 3 
--learning_rate 0.0001 
--loss mse 

Thanks for your efforts and I hope you can help me out!

**Comment by ricokoff** [comment]
[BTC_USD_CEX.csv](https://github.com/zhouhaoyi/Informer2020/files/9062381/BTC_USD_CEX.csv)
Great, here is my current dataset

**Comment by ricokoff** [comment]
@zhouhaoyi Could you find the cause of this problem? I tried several different (financial) datasets, but I keep getting the same problem. Maybe I am using the wrong parameters or I need to preprocess the data more? Please let me me know if you found something.

**Comment by zhouhaoyi** [comment]
We are checking on the data. Please watch our repo for updates.

**Comment by ricokoff** [comment]
@zhouhaoyi any updates? I need to get this working for my thesis.

**Comment by ricokoff** [comment]
BTW, in the prediction.pdf file I provided, you can see what I mean with the weird drop

**Comment by MountVoom** [comment]
The distribution of your training and testing sets of data is very different, and it will be difficult to predict.

**Comment by shayneliu** [comment]
我认为你在用这个模型之前先做下平稳性检验，或者只检验下相互依赖关系。如果时间序列前后的依赖关系很弱，多么好的模型估计都难有效果
